.github/workflows/install.yml

```diff
@@ -19,6 +19,20 @@ jobs:
       run: python3 -X tracemalloc -u -m unittest discover -v -f -s tests
     - name: Codecov
       run: curl -s https://codecov.io/bash | bash
+  build-win32:
+    runs-on: ubuntu-latest
+    container:
+      image: jeromerobert/wine-mingw64
+      options: --user root
+    steps:
+    - uses: actions/checkout@v2
+    - run: ./setup.py build
+    - run: HOME=/root wine python setup_win32.py bdist_msi
+    - run: HOME=/root wine python setup_win32.py bdist_zip
+    - uses: actions/upload-artifact@v2
+      with:
+        name: Windows installer
+        path: dist/*
```

Comment:
This creates one zip which is 68 MB in size, containing the MSI with 37MB and the ZIP with 35MB. Most users won't need both, what do you think about putting those in separate zips? (This would be a workaround to Github's current limitations.)

Reply:
Of course this can be merged as is, we can always change it later.